go/pkg/pwapi/api_team-accept-invite.go

```diff
@@ -4,13 +4,53 @@ import (
 	"context"
 
 	"pathwar.land/pathwar/v2/go/pkg/errcode"
+	"pathwar.land/pathwar/v2/go/pkg/pwdb"
 )
 
 func (svc *service) TeamAcceptInvite(ctx context.Context, in *TeamAcceptInvite_Input) (*TeamAcceptInvite_Output, error) {
-	if in == nil || in.TeamMemberID == 0 {
+	if in == nil || in.TeamInviteID == "" {
 		return nil, errcode.ErrMissingInput
 	}
 
+	userID, err := userIDFromContext(ctx, svc.db)
+	if err != nil {
+		return nil, errcode.ErrUnauthenticated.Wrap(err)
+	}
+
+	teamInviteID, err := pwdb.GetIDBySlugAndKind(svc.db, in.TeamInviteID, "team-invite")
+	if err != nil {
+		return nil, errcode.TODO.Wrap(err)
+	}
+
+	// check that invite exists
+	var teamInvite pwdb.TeamInvite
+	err = svc.db.
+		Where(&pwdb.TeamInvite{
+			ID:     teamInviteID,
+			UserID: userID,
+		}).
+		First(&teamInvite).
+		Error
+	if err != nil {
+		return nil, errcode.ErrTeamInviteDoesNotExist.Wrap(err)
```

Comment:
actually, `pwdb.GetIDBySlugAndKind` is the thing that checks if the entry exists, so if we hit an error here, it's an internal db error, not a "does not exists"

you can reuse this errcode on line 22 instead

and you can update the comment line 25